cmd/skaffold/app/cmd/flags.go

```diff
@@ -367,6 +367,15 @@ var flagRegistry = []Flag{
 		DefinedOn:     []string{"dev", "run", "debug", "deploy", "render", "build", "delete", "diagnose"},
 		IsEnum:        true,
 	},
+	{
+		Name:          "propagate-profiles",
+		Usage:         "Set to false to disable propagating profiles set by the '--profile' flag across config dependencies",
```

Comment:
nit. I personally find `Set to false to disable propagating profiles set by the '--profile' flag across config dependencies` a bit hard to read/understand.  One alternative (that is a bit long haha):
`Setting --propogate-profiles=false disables propagating profiles set by the '--profile' flag across config dependencies.  This makes the --profile flag value that is set used only for dependencies where a profile is not defined in the skaffold.yaml directly`

Reply:
done